src/v2/view-builder/views/idp/AuthenticatorIdPView.js

```diff
@@ -0,0 +1,20 @@
+import EnrollIdPBody from './EnrollAuthenticatorIdPBody';
+import ChallengeIdpBody from './ChallengeAuthenticatorIdPBody';
```

Comment:
nit: we can name this file and the one above `*IdPForm` instead of `*IdPBody`, since those extend `BaseForm`